### PR TITLE
Add auditable-information to user

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes none
| Related issues/PRs | none
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR extend the User with auditable information.

#### BC Breaks/Deprecations

SQL Statement provided.

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
...
 ## dev-develop
 
+### User-Table changed
+
+The user-table was extended with auditable information. Run following sql-statement to
+update the database schema.
+
+```bash
+ALTER TABLE se_users ADD changed DATETIME DEFAULT '1970-01-01 00:00:00' NOT NULL, ADD created DATETIME DEFAULT '1970-01-01 00:00:00' NOT NULL;
+ALTER TABLE se_users CHANGE created created DATETIME NOT NULL, CHANGE changed changed DATETIME NOT NULL;
```
Comment (danrot) [inline UPGRADE.md]: The fields for the changer and creator are missing here.

**Review by danrot — CHANGES_REQUESTED** [review]

File: UPGRADE.md (hunk 1 above)
Comment (wachterjohannes) [reply UPGRADE.md]: 👍 your right they are nullable and can be added by schema update - but i will add them also to the upgrade sql

**Comment by niklasnatter** [comment]
Is this still something we want to do? Could you rebase this onto the `2.x` branch? 🙂

**Comment by alexander-schranz** [comment]
ping @wachterjohannes

**Comment by alexander-schranz** [comment]
replaced by #6001

**Pull request closed without merging**